targets/csharp-unityv2/Testing/Tests/Utility/JsonUnitTests.cs

```diff
@@ -0,0 +1,286 @@
+
+        public static StructNumFieldTest Max = new StructNumFieldTest { SbyteValue = sbyte.MaxValue, ByteValue = byte.MaxValue, ShortValue = short.MaxValue, UshortValue = ushort.MaxValue, IntValue = int.MaxValue, UintValue = uint.MaxValue, LongValue = long.MaxValue, UlongValue = ulong.MaxValue, FloatValue = float.MaxValue, DoubleValue = double.MaxValue };
+        public static StructNumFieldTest Min = new StructNumFieldTest { SbyteValue = sbyte.MinValue, ByteValue = byte.MinValue, ShortValue = short.MinValue, UshortValue = ushort.MinValue, IntValue = int.MinValue, UintValue = uint.MinValue, LongValue = long.MinValue, UlongValue = ulong.MinValue, FloatValue = float.MinValue, DoubleValue = double.MinValue };
+        public static StructNumFieldTest Zero = new StructNumFieldTest();
+    }
+    internal class ObjOptNumFieldTest
+    {
+        public sbyte? SbyteValue { get; set; }
+        public byte? ByteValue { get; set; }
+        public short? ShortValue { get; set; }
+        public ushort? UshortValue { get; set; }
+        public int? IntValue { get; set; }
+        public uint? UintValue { get; set; }
+        public long? LongValue { get; set; }
+        public ulong? UlongValue { get; set; }
+        public float? FloatValue { get; set; }
+        public double? DoubleValue { get; set; }
+
+        public static ObjOptNumFieldTest Max = new ObjOptNumFieldTest { SbyteValue = sbyte.MaxValue, ByteValue = byte.MaxValue, ShortValue = short.MaxValue, UshortValue = ushort.MaxValue, IntValue = int.MaxValue, UintValue = uint.MaxValue, LongValue = long.MaxValue, UlongValue = ulong.MaxValue, FloatValue = float.MaxValue, DoubleValue = double.MaxValue };
+        public static ObjOptNumFieldTest Min = new ObjOptNumFieldTest { SbyteValue = sbyte.MinValue, ByteValue = byte.MinValue, ShortValue = short.MinValue, UshortValue = ushort.MinValue, IntValue = int.MinValue, UintValue = uint.MinValue, LongValue = long.MinValue, UlongValue = ulong.MinValue, FloatValue = float.MinValue, DoubleValue = double.MinValue };
+        public static ObjOptNumFieldTest Zero = new ObjOptNumFieldTest { SbyteValue = 0, ByteValue = 0, ShortValue = 0, UshortValue = 0, IntValue = 0, UintValue = 0, LongValue = 0, UlongValue = 0, FloatValue = 0, DoubleValue = 0 };
+        public static ObjOptNumFieldTest Null = new ObjOptNumFieldTest { SbyteValue = null, ByteValue = null, ShortValue = null, UshortValue = null, IntValue = null, UintValue = null, LongValue = null, UlongValue = null, FloatValue = null, DoubleValue = null };
+    }
+
+    internal class OtherSpecificDatatypes
+    {
+        public Dictionary<string, string> StringDict { get; set; }
+        public Dictionary<string, int> IntDict { get; set; }
+        public Dictionary<string, uint> UintDict { get; set; }
+        public Dictionary<string, Region> EnumDict { get; set; }
+        public string TestString { get; set; }
+    }
+
+    public class JsonFeatureTests : UUnitTestCase
+    {
+        //[UUnitTest]
```

Comment:
Please update SdkGenerator with my changes to V2.  This test is enabled and passing in that update.